pkg/flow/internal/import-source/import_source.go

```diff
@@ -0,0 +1,58 @@
+package importsource
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/grafana/agent/component"
+	"github.com/grafana/river/vm"
+)
+
+type SourceType int
+
+const (
+	FILE SourceType = iota
+	HTTP
+	GIT
```

Comment:
It's unidiomatic for constants to be all uppercase in Go; these should be `File`, `HTTP`, and `Git` respectively. (There's an exception for HTTP as it's an acronym).